package.json

```diff
@@ -291,7 +291,9 @@
     "@types/storybook__addon-centered": "^3.3.2",
     "@types/storybook__addon-knobs": "^5.0.2",
     "@types/storybook__react": "^4.0.2",
-    "babel-loader": "^8.0.6",
+    "acorn": "^7.1.1",
+    "babel-loader": "^8.1.0",
+    "babel-register": "^6.26.0",
```

Comment:
I don't think we need this. Locally just `acorn` and updating `babel-loader` did the trick

Reply:
Verified locally on Linux that it's not needed 👍 Let's let the Jenkins run finish and I can update / let Travis run / and we're good for merge 😄

Reply:
Fixed with https://github.com/brave/brave-core/pull/5013/commits/e26d081e25dabf6c282a2425cec06f0dfe9dbde2